packages/eslint-plugin-qwik/src/useMethodUsage.ts

```diff
@@ -92,21 +62,21 @@ export const useMethodUsage: Rule.RuleModule = {
               }
               context.report({
                 node,
-                messageId: 'use-wrong-function',
+                messageId: 'useWrongFunction',
               });
               return;
             case 'FunctionDeclaration':
               if (!parent.id?.name.startsWith('use')) {
                 context.report({
                   node,
-                  messageId: 'use-wrong-function',
+                  messageId: 'useWrongFunction',
                 });
               }
               return;
             default:
               context.report({
                 node,
-                messageId: 'use-not-root',
+                messageId: 'useWrongFunction',
```

Comment:
eslint errors are always in kebab-case, not camelCase.

Reply:
Yes. It is still in kebab-case in index.ts. If you look at the other files (e.g. preferClasslist.ts) you will see that `messageId` is always in camelCase. So I changed `messageId` syntax to be like in the other files. I think it's good also because using camelCase for messageId allows for typescript autocomplete :).